src/invidious/yt_backend/connection_pool.cr

```diff
@@ -59,9 +59,14 @@ struct YoutubeConnectionPool
   end
 end
 
-def make_client(url : URI, region = nil)
+def make_client(url : URI, region = nil, force_resolve : Bool = true)
```

Comment:
Shouldn't this be set to `force_resolve : Bool = false`? We do not want force_resolve by default.